From 2499f29921855536d8e862003ac1781781c7dc3a Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Sat, 13 Jun 2020 00:16:34 -0700
Subject: [PATCH 1379/1851] misc: ai-engine: change driver name

commit b4fd9f6387696afcbb967597020a70f49d2ed1c7 from
https://github.com/Xilinx/linux-xlnx.git

Change the driver name to xilinx-ai-engine not to conflict to the uio AI
engine driver name.

Signed-off-by: Wendy Liang <wendy.liang@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/misc/xilinx-ai-engine/ai-engine-dev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/xilinx-ai-engine/ai-engine-dev.c b/drivers/misc/xilinx-ai-engine/ai-engine-dev.c
index e2c4ee1b41c1..4fe00bbd4aa5 100644
--- a/drivers/misc/xilinx-ai-engine/ai-engine-dev.c
+++ b/drivers/misc/xilinx-ai-engine/ai-engine-dev.c
@@ -390,7 +390,7 @@ static struct platform_driver xilinx_ai_engine_driver = {
 	.probe			= xilinx_ai_engine_probe,
 	.remove			= xilinx_ai_engine_remove,
 	.driver			= {
-		.name		= "xilinx-aiengine",
+		.name		= "xilinx-ai-engine",
 		.of_match_table	= xilinx_ai_engine_of_match,
 	},
 };
-- 
2.31.1

